test/main_test.js

```diff
@@ -86,3 +91,14 @@ function checkExists(path) {
     `${path} should exist`
   );
 }
+
+function checkEnvInNinja(path) {
+  let read = new Promise((resolve, reject) => {
```

Comment:
I was thinking that you could even do this in your `env/configure.js`. Does that sound good?